queue_processors/autoreduction_processor/post_process_admin.py

```diff
@@ -203,10 +203,14 @@ def reduce(self):
             if self.message.description is not None:
                 logger.info("DESCRIPTION: %s", self.message.description)
 
+            def create_log_path(file_name, log_directory):
+                """Create log file and place in reduction_log_directory
+                :param (string) file name and extension type"""
```

Comment:
Nitpick: Could this say `filename with extension`

Since the word type makes me think we have a class type that takes a filename and extension rather than a string